docs/api/airgun.views.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`airgun.views`
+===================
+
+.. automodule:: airgun.views
+   :members:
```

Comment:
Pls insert existing views here.

Reply:
See comment above. Again, I can change the structure so airgun.views.rst contains all of them.

Reply:
Done this as well. Moved from individual files into one.